### PR TITLE
Link CoreGraphics and Foundation frameworks on Apple platforms

### PR DESCRIPTION
**Pull request by matteblair**
This allows alfons to be built for Apple targets without requiring another target to link these frameworks.

### CODE REVIEW

**Review by tallytalwar — APPROVED** [review]
Looks good.

**Comment by tallytalwar** [comment]
@hjanetzek this looks good to me and tested on osx. Just waiting on your approval to merge.

**Comment by matteblair** [comment]
ping

**Pull request merged**
